### DISCUSSION TITLE
Standard approach for all emails?

### DISCUSSION THREAD

**Discussion by yowzadave** [supabase/supabase#5806] [Questions]
I'm using SendGrid to send emails for various purposes; is it possible to use my own custom workflow for ALL emails, including the built-in events that trigger emails? Or, alternatively, to extend the built-in email system with my own custom emails?

Currently there are a select set of emails that are triggered by supabase; these are sent via the method configured in the web UI, or are caught by the Inbucket when developing locally with the CLI. If I have a need to send any other emails, it seems like the process for composing/sending those other emails is entirely separate. I would prefer to be able to send all of these emails via my custom method (since the standard email templates seem to be limited and don't support i18n).

Can I disable the built-in mailing method and replace it with something custom?

**Comment by GaryAustin1** [comment]
You might find this discussion interesting or want to participate:
https://github.com/supabase/gotrue/issues/304#issuecomment-1022115983

**Reply by yowzadave** [reply]
Thanks! Yes, it sounds like those complaints and potential resolutions are similar to mine.
